styles.js

```diff
@@ -1,4 +1,6 @@
-import { createGlobalStyle } from "styled-components";
+import styled, { createGlobalStyle } from "styled-components";
+
+import { Roboto } from "next/font/google";
```

Comment:
You are currently not using this font. Therefore, you should remove this line of code:

```suggestion
```